### PR TITLE
Migrate to AndroidX

### PR DESCRIPTION
**Pull request by ar-arvind**
Migrated the project to AndroidX library.

### CODE REVIEW

File: Android/app/src/main/java/com/example/samplestickerapp/EntryActivity.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import android.content.Intent;
 import android.os.AsyncTask;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
```
Comment (simonzhexu) [inline Android/app/src/main/java/com/example/samplestickerapp/EntryActivity.java]: please do a reformat, and that will move this line to the bottom.

Comment (ar-arvind) [reply Android/app/src/main/java/com/example/samplestickerapp/EntryActivity.java]: Code reformatted!

**Comment by ar-arvind** [comment]
@simonzhexu I reformatted the code. Please check and merge this pull request.

**Comment by MarvinJWendt** [comment]
Hey @ar-arvind, as it seems @simonzhexu is busy at the moment. Please be patient he will definitely take a look as soon as he finds some time :)

**Comment by ar-arvind** [comment]
@MarvinJWendt  👍

**Comment by simonzhexu** [comment]
Sorry for the delay, looks good. Thanks.

**Comment by ar-arvind** [comment]
@simonzhexu Thank you 👍

**Pull request merged**